kfs-cam/src/main/java/org/kuali/kfs/module/cab/businessobject/PurchasingAccountsPayableProcessingReport.java

```diff
@@ -23,6 +23,8 @@
 public class PurchasingAccountsPayableProcessingReport extends GeneralLedgerEntry {

```

Comment:
It has class casting issue still as I tried to make edu BO. In PurApReportLookupableHelperServiceImpl which has a few methods binding BO type in the method signature, for example, List<PurchasingAccountsPayableProcessingReport> updatePurApReportListByPurApDocs(List<PurchasingAccountsPayableProcessingReport> purApReportList, Map<String, Integer> purApDocNumberMap).  

We have to either change method signature to generic type in org service code or have to redefine those methods in edu service with edu BO type. Then, it can incur more code changes and override org methods which are not necessary. Though modifying POJO in org might need additional code merge for future upgrade, it could be less problematic to avoid class casting issue when upgrade from what I saw. But I'm open to either way. Please let me know how you'd like this to proceed.